### PR TITLE
Show usage

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description

Adds the usage view to all org members (previously only owners could see usage)

## Related Issue(s)

Fixes WEB-314

## How to test

Join https://se-usage.preview.gitpod-dev.com/orgs/join?inviteId=71788c56-a3bb-4de1-913c-ca81de001327

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - se-usage</li>
	<li><b>🔗 URL</b> - <a href="https://se-usage.preview.gitpod-dev.com/workspaces" target="_blank">se-usage.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options:

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [x] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

/hold

### CODE REVIEW

File: components/server/src/iam/iam-session-app.spec.ts [hunk 1]
```diff
@@ -196,7 +196,7 @@ class TestIamSessionApp {
         expect(result.body?.message).to.equal("OIDC client config id is missing");
     }
 
-    @test.only public async testSessionRequest_createUser_removes_admin() {
```
Comment (svenefftinge) [inline components/server/src/iam/iam-session-app.spec.ts:199]: @AlexTugarev was this a leftover from developing?

Comment (AlexTugarev) [reply components/server/src/iam/iam-session-app.spec.ts:199]: nice catch!
thanks!

**Review by AlexTugarev — APPROVED** [review]
LGTM! 
Works as advertised!

**Comment by AlexTugarev** [comment]
/unhold

tide is striking

**Pull request merged**